lesson_4_js_basics.js

```diff
@@ -0,0 +1,58 @@
+/* 1. Написати функцію, яка у випадку якщо типи двох аргументів string поверне їх зконкатенований вираз,
+якщо типи двох аргументів number - поверне їх помножене значення
+та у будь якому випадку не співпадіння поверне текст There are no type matches for operation */
+
+function typeChecker(arg1, arg2) {
+    let result; 
+if (typeof arg1 === 'string'  && typeof  arg2 === 'string'){
+    result = arg1 + ' ' + arg2; 
+}
+else if (typeof arg1 === 'number'  && typeof  arg2 === 'number'){
+    result = arg1 + arg2; 

```

Comment:
```suggestion
    result = arg1 * arg2; 

```
якщо типи двох аргументів number - поверне їх помножене значення